azure-pipelines.yml

```diff
@@ -97,10 +105,16 @@ jobs:
       ATLAS: None
       ACCELERATE: None
       CC: /usr/bin/clang
+  # wait until after dev build of NumPy to pip
+  # install matplotlib to avoid pip install of older numpy
+  - script: python -m pip install matplotlib
+    displayName: 'Install matplotlib before refguide run'
```

Comment:
discovered as a result of probing the OpenBLAS version; our distributed wheels won't match 0.3.7.dev just yet